From ac7d039201d145590bba1ebdce684f088afeefaf Mon Sep 17 00:00:00 2001
From: Jakub Jursa <jakub.jursa1@gmail.com>
Date: Fri, 11 May 2018 15:00:07 +0200
Subject: [PATCH] Fixes starting freezer-scheduler with --no-daemon

When using flag --no-daemon, freezer-scheduler --no-daemon start fails with
AttributeError: 'NoDaemon' object has no attribute 'restart'
(freezer_scheduler.py: L251). This patch adds missing dummy methods for reload
and restart actions.

Change-Id: I72a0a4f68503b7c143cbe6c6f13cfbdc08e6ccbb
---
 freezer/scheduler/daemon.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/freezer/scheduler/daemon.py b/freezer/scheduler/daemon.py
index 9e5562d..1a7076e 100644
--- a/freezer/scheduler/daemon.py
+++ b/freezer/scheduler/daemon.py
@@ -115,6 +115,12 @@ class NoDaemon(object):
     def status(self):
         pass
 
+    def restart(self):
+        pass
+
+    def reload(self):
+        pass
+
 
 class Daemon(object):
     """
-- 
2.7.4

